dobackup.py

```diff
@@ -68,7 +75,8 @@ def do_backup():
     for i in range(ungotten, count + 1):
         uid = getUIDForMessage(svr, i)
         print "Downloading %d/%d (UID: %s)" % (i, count, uid)
-        downloadMessage(svr, i, uid + '.eml')
+        filepath = save_folder_path() + uid + '.eml'
```

Comment:
Can you use os.path.join to combine the folder path with the rest? That's better practice in general and should work even if they don't put a trailing slash on DOBACKUP_SAVE_FOLDER